docs-src/docs/guides/consul.md

```diff
@@ -1,112 +1,2 @@
-    {
-        "name": "EXTERNAL_URL",
-        "value": "&lt;url&gt;"
-     }
-]
-</code></pre>
-<br />
-Replace <em>&lt;url&gt;</em> with the URL you copied in step 2 of the previous section, and then save the file.</li>
-  <li>At the command line, type the following command to create a new deploy called <strong>consul</strong>:
-    <ul>
-      <li class="command"><strong>l0 deploy create Consul.Dockerrun.aws.json consul</strong></li>
-    </ul><br />
-  You will see the following output:
-<pre class="code"><code>DEPLOY ID  DEPLOY NAME  VERSION
-1consul:1  consul       1
-</code></pre>
-  </li>
-</ol>
-
-## Part 3: Create the service
-Now that you've created an environment, load balancer, and deploy, you can create a service to bring these elements together.
-
-**To create the service:**
-<ol>
-  <li>At the command line, type the following command to create a new service called **consul**:
-    <ul>
-      <li class="command">**l0 service create --loadbalancer demo:consullb demo consulsvc consul:latest**</li>
-    </ul><br />
-    You will see the following output:
-<pre class="code"><code>SERVICE ID  SERVICE NAME  ENVIRONMENT  LOADBALANCER  DEPLOYS      SCALE
-1consul     consulsvc     demo         consullb      consul:1     0/1</code></pre>
-  </li>
-  <li>Wait several minutes for the service to be provisioned. You can check the status of the service creation by running the following command:
-  <ul>
-    <li class="command">**l0 service get consulsvc**</li>
-  </ul><br />
-  When the service has finished provisioning, you will see the following output:
-```
-SERVICE ID  SERVICE NAME  ENVIRONMENT  LOADBALANCER  DEPLOYS      SCALE
-1consul     consulsvc     demo         consullb      consul:1     1/1
-```
-  </li>
-</ol>
-
-## Part 4: Scale the Consul service
-Consul is a scalable application. For added reliability, we recommend that you scale the Consul service to size 3.
-
-**To scale the service:**
-
-<ol>
-  <li>At the command line, type the following to scale the consul service to size 3:
-    <ul>
-      <li class="command"><strong>l0 service scale demo:consulsvc 3</strong></li>
-    </ul>
-  </li>
-
-  <li>Wait several minutes for the service to scale. You can check the status of the service by running the following command:
-  <ul>
-    <li class="command"><strong>l0 service get consul</strong></li>
-  </ul><br />
-  When the Service has finished scaling, you will see the following output:
-<pre class="code"><code>SERVICE ID  SERVICE NAME  ENVIRONMENT  LOADBALANCER  DEPLOYS      SCALE
-1consul     consulsvc     demo         consullb      consul:1     3/3
-</code></pre>
-  </li>
-</ol>
-
-## Additional steps: Configure Layer0 services
-In order to for Layer0 services to use consul, the task definitions for those services must be configured. The next deployment guide in this series ([Guestbook with Consul](/guides/guestbook_consul)) contains an example of these configurations.
+!!! warning "Under Construction"
```

Comment:
I realize @zpatrick suggested otherwise, but I don't think we should have WIP and/or 'Under Construction' in our published docs.  The state of `layer0-examples` should include the samples that we want/need first and foremost and the docs should then reflect that, not the other way around.